js/Game.js

```diff
@@ -108,7 +108,7 @@ class Game {
     } else {
       alert('You got the WRONG answer')
```

Comment:
We'll need to remember to change these to actual pop ups or indications tomorrow.